pkg/filter/filter_test.go

```diff
@@ -95,6 +95,24 @@ func TestFilterVulnerabilities(t *testing.T) {
 				},
 			},
 		},
+		{
+			"filters out vulnerabilities that have bad status",
+			7.0,
+			"test-vuln-.*",
+			[]domain.Vulnerability{
+				domain.Vulnerability{
+					ID:          "bad-vuln-1",
+					CvssV2Score: 8.0,
+					Status:      "invulnerable",
+				},
+				// domain.Vulnerability{
```

Comment:
Task: This should be uncommented or removed.

Reply:
fixed